qa/suites/rgw/notifications/tasks/test_first_rabbitmq.yaml

```diff
@@ -0,0 +1,11 @@
+tasks:
+- tox: [ client.0 ]
+- rabbitmq:
+    client.0:
+      rabbitmq_version: 3.8.9
+- notification-tests:
+    client.0:
+      force-branch: wip-rgw-rabbitmq
+      git_remote: https://github.com/TRYTOBE8TME/
+      extra_attr: ["AMQP test"]
```

Comment:
do you update the kafka file with its "Kafka test" attribute?
what about "HTTP tests", and basic tests that don;t use an endpoint?

Reply:
Will do, thanks.